iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt

```diff
@@ -301,74 +303,74 @@ class MapboxMapFragment : Fragment() {
     @SuppressLint("MissingPermission")
     private fun setupMap(mapView: MapView) {
         mapView?.getMapAsync { map ->
-            map.setStyle("mapbox://styles/dchiles/cj3nxjqli000u2soyeb947f7s") {
-            }
-        }
-        val initZoomAmount = 0.2
-        val pos = CameraPosition.Builder()
-                .target(LatLng(Geo.MAN_LAT, Geo.MAN_LON))
-                .zoom(defaultZoom - initZoomAmount)
-                .build()
-
-        mapView.getMapAsync { map ->
-            val hasLocationPermission = context
-                    ?.let { PermissionManager.hasLocationPermissions(it) }
-                    ?: false
-
-            /*val locationLayerPlugin: LocationLayerPlugin? = if (BuildConfig.MOCK) {
-                val engine = LocationProvider.MapboxMockLocationSource()
-                engine.activate()
-                engine.requestLocationUpdates()
-                val plugin = LocationLayerPlugin(mapView, map)
-                plugin.locationEngine = engine
-                plugin
-            } else if (hasLocationPermission) {
-                LocationLayerPlugin(mapView, map)
-            } else {
-                null
-            }
+            val style = Style.Builder().fromUri("asset://map/style.json").withSource(VectorSource("composite", "asset://map/map.mbtiles"))
```

Comment:
The font glyphs folder is also required or the map won't render. (its within the same folder as the map and style in the data repo)

Reply:
Yeah, just checked in